### PR TITLE
13.0 mig mail activity team

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Work done:
- Change version to 13.0.1.0.0
- remove @api.multi
- Do views installable (fix xpath to variables which does not exist anymore)
- Replace sudo(user) by with_user(user)
Fix on tests:
- Do not user field days() as it’s not used since
- Remove last test done inside function test_activity_onchanges() because it’s not coherent with functionality _check_team_and_user()

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
@jbeficent, @ageficent, @mreficent .. module depends on mail_activity_board which is also waiting to be reviewed #451 . But in case you want to review code/functionality will be  appreciated. Thanks

File: mail_activity_team/models/mail_activity.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            # to member_ids it's not taken account
+            # To not be blocked we must add it to constraint condition
+            if (
+                activity.user_id.id != SUPERUSER_ID
+                and activity.team_id
+                and activity.user_id
+                and activity.user_id not in self.team_id.member_ids
+            ):
+                raise ValidationError(
+                    _("The assigned user is not member of " "the team.")
```
Comment (MiquelRForgeFlow) [inline mail_activity_team/models/mail_activity.py]:
```suggestion
                    _("The assigned user is not member of the team.")
```

**Comment by pegonzalezspesol** [comment]
thanks @mreficent for the fast revision!

**Comment by pegonzalezspesol** [comment]
@angelmoya please review

File: mail_activity_team/README.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/social <https://github.com/OCA/social/tree/12.0/mail_activity_team>`_ project on GitHub.
```
Comment (angelmoya) [inline mail_activity_team/README.rst]: Change all 12.0 to 13.0

File: mail_activity_team/models/mail_activity.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                self.user_id = self.team_id.user_id
+            elif len(self.team_id.member_ids) == 1:
+                self.user_id = self.team_id.member_ids
+            else:
+                self.user_id = self.env["res.users"]
+        return res
+
+    @api.constrains("team_id", "user_id")
+    def _check_team_and_user(self):
+        for activity in self:
```
Comment (angelmoya) [inline mail_activity_team/models/mail_activity.py]: for ... if can be replace by filtered

**Review by angelmoya — APPROVED** [review]
Small remarks, but LGTM

**Review by antoniocanovas — APPROVED** [review]
Functional Review Ok

File: mail_activity_team/models/mail_activity.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        # SUPERUSER is inactive and then even if you add it
+        # to member_ids it's not taken account
+        # To not be blocked we must add it to constraint condition
+        for _activity in self.filtered(
+            lambda a: a.user_id.id != SUPERUSER_ID
+            and a.team_id
+            and a.user_id
+            and a.user_id not in a.team_id.member_ids
+        ):
+            raise ValidationError(_("The assigned user is not member of " "the team."))
```
Comment (MiquelRForgeFlow) [inline mail_activity_team/models/mail_activity.py]:
```suggestion
            raise ValidationError(_("The assigned user is not member of the team."))
```

**Pull request merged**